mm2src/coins/lp_coins.rs

```diff
@@ -1620,6 +1620,11 @@ pub trait AddrToString {
     fn addr_to_string(&self) -> String;
 }
 
+/// A trait for converting an address into a string suitable for display in logs, errors, or messages.
+pub trait DisplayAddress {
+    fn display_address(&self) -> String;
+}
```

Comment:
I think this name is more idiomatic....

```suggestion
pub trait ToAddressStr {
    fn to_address_str(&self) -> String;
}
```

Reply:
Display trait has another purpose (and we have it in dev, I just moved it from chain specific file). 
We have `AddrToString` trait, it was added not so long time ago https://github.com/KomodoPlatform/komodo-defi-framework/issues/2343

https://github.com/KomodoPlatform/komodo-defi-framework/blob/3556fd6d4985c2792b8e4c14e027909a9b7eaf3f/mm2src/coins/lp_coins.rs#L1613-L1621